azure-pipelines.yml

```diff
@@ -73,11 +72,16 @@ jobs:
   # matches our MacOS wheel builds -- currently based
   # primarily on file size / name details
   - script: |
-      wget "https://3f23b170c54c2533c070-1c8a9b3114517dc5fe17b7c3f8c63a43.ssl.cf2.rackcdn.com/openblas-v0.3.3-186-g701ea883-macosx_10_9_intel-gf_1becaaa.tar.gz"
-      tar -zxvf openblas-v0.3.3-186-g701ea883-macosx_10_9_intel-gf_1becaaa.tar.gz
+      wget -O openblas-mac.tar.gz "https://www.dropbox.com/s/64u1h9sm7uq0vrd/openblas-mac.tar.gz?dl=0"
+      tar -zxvf openblas-mac.tar.gz
       # manually link to appropriate system paths
       cp ./usr/local/lib/* /usr/local/lib/
       cp ./usr/local/include/* /usr/local/include/
+      # OpenBLAS build is more portable now
+      # strictly enforce lack of runtime deps
+      # libgfortran & libquadmath should be statically linked
+      ! otool -L ./usr/local/lib/libopenblasp-r0.3.5.dev.dylib | grep -q 'libgfortran'
+      ! otool -L ./usr/local/lib/libopenblasp-r0.3.5.dev.dylib | grep -q 'libquadmath'
```

Comment:
These one-liners should be made slightly more robust. They are working as intended for now, but if the object file being parsed doesn't exist the return code inversion specified by `!` will allow for silent failure.

Probably have to bite the bullet and use `if` statements in bash or whatever.